tools/chouette/initialize-chouette.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+echo "1. Create /opt/chouette/ folder"
+if [ ! -d "/opt/chouette/" ]; then
+
+  sudo mkdir /opt/chouette
```

Comment:
```suggestion
  sudo mkdir -p /opt/chouette
```
-p switch creates whole path structure, but also as a side effect does not complete using an error if directory already exists.